runtime/src/iree/builtins/ukernel/arch/x86_64/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@
 
 iree_add_all_subdirs()
 
+file(GLOB _GLOB_X_H LIST_DIRECTORIES false RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} CONFIGURE_DEPENDS *.h)
```

Comment:
Note @ScottTodd , I was initially confused by this line, which creates a `_GLOB_X_H` variable that's just unused. But it is created by `bazel_to_cmake` in the lowering of `glob`, and then it goes unused because `exports_files` is implemented as `pass`.

Reply:
I wonder if Python has a concept of macros that could be used instead of functions for the empty-implementation entry points which are currently functions doing `pass`, so that they could skip evaluating their arguments.

Reply:
I think @GMNGeoffrey added that file glob stuff :P

Reply:
Anyway. I say we merge this now?

Reply:
Indeed I did. I kind of think that the entire implementation of bazel->cmake should be constructing objects and then only writing at the end rather than concatenating strings. Then this sort of thing wouldn't happen. This particular behavior is not awesome because CMake globs are not awesome (which is why we have `enforce_glob`). Although since implementing that I think I concluded that CMake glob was not as bad as everyone made it out to be as long as you use `ninja`, which supports `CONFIGURE_DEPENDS`. Ben was seeing some slowness that he attributed to the globbing, but I think it may have been a red herring. Or maybe it was just a matter of scale.

But long aside aside, these globs appear to mostly be globbing like one file. I think we can just list that explicitly or use `enforce_glob` if we're worried about it falling out of sync.

It's also usually better to use `filegroup` rather than `exports_files`. In the case of schemas, for instance, the only place those files are included, they are included together into another filegroup. Should we just make this one a filegroup?

https://github.com/openxla/iree/blob/main/runtime/src/iree/builtins/ukernel/BUILD.bazel#L118

Reply:
oh right, that is the right way. working on it.

Reply:
Alright, I just pushed a change to this PR, implementing this.

And I've also sent a LLVM diff to address a similar issue in the LLVM overlay,
https://reviews.llvm.org/D152491

So, putting this together, this command actually succeeds:

```
bazel build --incompatible_no_implicit_file_export //runtime/src/iree/builtins/ukernel/...
```